packages/react-search-ui-views/src/PagingInfo.js

```diff
@@ -10,7 +10,7 @@ function PagingInfo({ className, end, searchTerm, start, totalResults }) {
       <strong>
         {start} - {end}
       </strong>{" "}
-      out of <strong>{totalResults}</strong> for: <em>{searchTerm}</em>
```

Comment:
I don't think we should delete the `for` word completely - only if `searchTerm` is missing. So we should really be wrapping this in a conditional. Does that make sense?

So if a `searchTerm` is present, the string should still be:
```Showing X - Y out of Z for: FooBar```
and if not:
```Showing X - Y out of Z:```

@JasonStoltz / @yakhinvadim Feel free to let me know if you disagree and I'll update my review :)

Reply:
Yes but one small correction...

So if a searchTerm is present, the string should still be:

```
Showing X - Y out of Z for: FooBar
```

and if not:

```
Showing X - Y out of Z
```

No semi-colon in the second case, right?

Reply:
Oh yeah, no colon. I'll update my comment!

Reply:
Cool, I'll make the required changes

Reply:
@JasonStoltz @constancecchen Pushed some commits that address this. Do let me know if you'd like it to be done in a different way.